base/system/winlogon/lang/cs-CZ.rc

```diff
@@ -44,14 +54,14 @@ BEGIN
     IDS_RUNNINGLOGONSCRIPTS "Skripty po přihlášení..."
     IDS_LOADINGYOURPERSONALSETTINGS "Nahrávání osobního nastavení..."
     IDS_CLOSINGNETWORKCONNECTIONS "Ukončení síťových připojení..."
-    IDS_REACTOSISRESTARTING "Restarting..."
-    IDS_REACTOSISSHUTTINGDOWN "Shutting down..."
+    IDS_REACTOSISRESTARTING "Restartování..."
+    IDS_REACTOSISSHUTTINGDOWN "Vypínání..."
     IDS_PREPARETOSTANDBY "Příprava k přechodu do úsporného režimu..."
     IDS_PREPARETOHIBERNATE "Příprava k přechodu do režimu hibernace..."
     IDS_SAVEYOURSETTINGS "Ukládání osobního nastavení..."
     IDS_REACTOSISSTARTINGUP "ReactOS se spouští..."
     IDS_TIMEOUTSHORTFORMAT "%02d:%02d:%02d"
-    IDS_TIMEOUTLONGFORMAT "%d days"
+    IDS_TIMEOUTLONGFORMAT "%d dny/dní"
```

Comment:
The two forms are for when the number is <= 5 or greater (or something similar)?

Reply:
@HBelusca Yes. If the number is between 1 and 4, the proper form is `dny`, above 5 it is `dní`. This is common trait in Slavic languages.

Reply:
Would something like `%d dny(-í)` fit better?

Reply:
Current russian translation in our winlogon is: `%d дней` - but for some reason Windows XP has this left untranslated:

![image](https://github.com/reactos/reactos/assets/578406/a697dc92-c652-4568-96da-7945f3ea91df)

Reply:
Hmmm.... I would agree with @Venca24 here, in that it's not that natural. In french (where we just have singular/plural) it may be better to have `"%d jour(s)"` instead of either `"%d jour"` (always singular) or`"%d jours"` (always plural)

Reply:
If we want to use this form, it must be updated in all places at once (and in another PR).